README.md

```diff
@@ -477,24 +477,16 @@ logger.debug("Hello {}!", "world");
 ### Time
 
 openHAB internally makes extensive use of the `java.time` package.
-openHAB-JS exports the excellent [JS-Joda](#https://js-joda.github.io/js-joda/) library via the `time` namespace, which is a native Javascript port of the same API standard used in Java for `java.time`.
+openHAB-JS exports the excellent [JS-Joda](#https://js-joda.github.io/js-joda/) library via the `time` namespace, which is a native JavaScript port of the same API standard used in Java for `java.time`.
 Anywhere that a native Java `ZonedDateTime` or `Duration` is required, the runtime will automatically convert a JS-Joda `ZonedDateTime` or `Duration` to its Java counterpart.
 
-#### openHAB-JS extensions to JS-Joda

```

Comment:
I don't have a problem with that except everything I've implemented is outside of JS-Joda and not extensions to it. So that heading as written is incorrect.

But my branch doesn't have the millisFromNow. I suspect that was added after I opened this PR. I removed that heading because all that was under it was "Examples:" on down. There wasn't anything there to preserve and the bulk of the content was no longer references to JS-Joda.

Reply:
Okay, leave it as it is.

I will have a look what happens when this gets merged. If `millisFromNow` is removed, I will add it back.